src/ImGui.NET/ImGuiNative.cs

```diff
@@ -239,6 +225,23 @@ public static unsafe class ImGuiNative
         [DllImport(cimguiLib)]
         public static extern float igGetItemsLineHeightWithSpacing();
 
+        // Columns
```

Comment:
Ok, no problem. The original ordering was probably quite random. I used a tool to copy/paste all of the public exports from cimgui and then translated them all into PInvokes from there. So they are in whatever order the tool I exported them from used...